src/riak_kv_eleveldb_backend.erl

```diff
@@ -47,13 +47,12 @@
          is_empty/1,
          status/1,
          callback/3]).
-
 -export([data_size/1]).
 
--compile({inline, [
-                   to_object_key/2, from_object_key/1,
-                   to_index_key/4, from_index_key/1
-                  ]}).
+% -compile({inline, [
```

Comment:
Just a development change so I can trace these functions. Does it really improve performance?